app/refactor/views/p5/unloadingpermission/components/table_1/row_3.scala.xml

```diff
@@ -58,126 +58,57 @@
-                    <fo:table-row height="7mm"
-                                  border-bottom-style="solid" border-bottom-width="1pt" border-bottom-color="black"
-                    >
-                        <fo:table-cell  margin-left="0.5mm">
-                            <fo:block-container overflow="hidden" margin-top = "0.8mm">
-                                <fo:block>@(seals)</fo:block>
-                            </fo:block-container>
-                        </fo:table-cell>
-                    </fo:table-row>
-
-                    <!-- Previous document row -->
-
-                    <fo:table-row height="3mm">
-                        <fo:table-cell  margin-left="0.5mm">
-                            <fo:block-container overflow="hidden" margin-top = "0.8mm">
-                                <fo:block><fo:inline font-weight="bold">Previous document</fo:inline> [1201]</fo:block>
-                            </fo:block-container>
-                        </fo:table-cell>
-                    </fo:table-row>
-                    <fo:table-row height="7mm" border-bottom-style="solid" border-bottom-width="1pt" border-bottom-color="black">
-                        <fo:table-cell  margin-left="0.5mm">
-                            <fo:block-container overflow="hidden" margin-top = "0.8mm">
-                                <fo:block>@(previousDocument)</fo:block>
-                            </fo:block-container>
-                        </fo:table-cell>
-                    </fo:table-row>
-
-                    <!-- Transport document row -->
-
-                    <fo:table-row height="3mm">
-                        <fo:table-cell  margin-left="0.5mm">
-                            <fo:block-container overflow="hidden" margin-top = "0.8mm">
-                                <fo:block><fo:inline font-weight="bold">Transport document</fo:inline> [1205]</fo:block>
-                            </fo:block-container>
-                        </fo:table-cell>
-                    </fo:table-row>
-                    <fo:table-row height="7mm" border-bottom-style="solid" border-bottom-width="1pt" border-bottom-color="black">
-                        <fo:table-cell  margin-left="0.5mm">
-                            <fo:block-container overflow="hidden" margin-top = "0.8mm">
-                                <fo:block>@(transportDocument)</fo:block>
-                            </fo:block-container>
-                        </fo:table-cell>
-                    </fo:table-row>
+                <fo:table-column column-number="1" column-width="59mm"/>
+                <fo:table-column column-number="2" column-width="59mm" />
+                <fo:table-column column-number="3" column-width="59mm" />
+                <fo:table-body border-bottom-style="solid" border-bottom-width="1pt" border-bottom-color="black">
+
+                    <!--   Departure means rows-->
+
+                    @departureTransportMeans.map {
```

Comment:
Why not just make departureTransportMeans a Seq[String] and do the zipWithIndex here?